
MIN_APPROACH_ERROR : 0.05
MIN_GRIP_ERROR : 0.04
MAX_GRIP_LOAD : 0.2
APPROACH_X_OFFSET: -0.15
APPROACH_Y_OFFSET: -0.02
APPROACH_Z_OFFSET: 0.03

GRASP_X_OFFSET : -0.025
GRASP_Y_OFFSET : -0.02
GRASP_Z_OFFSET : 0.01
